React/CoreModules/RCTAlertManager.mm

```diff
@@ -199,7 +199,7 @@ - (void)invalidate
   [_alertControllers addObject:alertController];
 
   dispatch_async(dispatch_get_main_queue(), ^{
-    [alertController show:YES completion:nil];
+    [RCTPresentedViewController() presentViewController:alertController animated:animated completion:completion]; // [TODO(macOS GH#774) When using Scenes, we must present the alert from a view controller associated with a window in the Scene
```

Comment:
`animated` and `completion` aren't defined anywhere.

Reply:
Should we move the call to `presentViewController:animated:completion:` to inside RCTAlertController.m?

Reply:
Thanks, copy paste error. Yea we can do a check for if it's in the scene first.